src/api/fetchAPI.ts

```diff
@@ -0,0 +1,7 @@
+export interface fetchApi {
```

Comment:
This is an important point of the application and needs some documentation. What are the parameters for each function and what does it return?